docs/UserGuide.md

```diff
@@ -144,24 +147,41 @@ Parameters:
 Restrictions:
 * At least one of the optional fields must be provided.
 * The search is case-insensitive. e.g `n/hans` will match `Hans`.
-* The order of the keywords does not matter. e.g. `s/chef n/Hans` and `n/Hans s/chef` are valid inputs.
 * Allows partial matching of keywords e.g. `n/Han` will match `Hans`.
-* Volunteers matching **at least one** NAME keyword will be returned (i.e. `OR` search).
-  e.g. `n/Hans n/Bo` will return `Hans Gruber`, `Bo Yang`.
-* Volunteers matching **both** SKILL keywords will be returned (i.e. `AND` search).
-  e.g. `s/chef s/boxer` will return volunteers that have skills `chef` and `boxer`.
+* Both the volunteer name and corresponding skills can be searched.
+* The order of the keywords does not matter. e.g. `s/chef n/Hans` and `n/Hans s/chef` are valid inputs.
+
+Common cases:
+* Case 1: user searches for **one** name
+  * Volunteers who have that NAME keyword will be returned
+* Case 2: user searches for **more than one** names
+  * Volunteers who have **at least one** of the NAME keywords will be returned (i.e. `OR` search).
+    e.g. `n/Hans n/Bo` will return `Hans Gruber`, `Bo Yang`.
+* Case 3: user searches for **one** skill
+  * Volunteers who have that SKILL keyword will be returned
+* Case 4: user searches for **more than one** skills
+  * Volunteers who have **both** SKILL keywords will be returned (i.e. `AND` search).
+    e.g. `s/intelligent s/smart` will return volunteers that have both skills `intelligent` and `smart`.
+* Case 5: user searches for **one** name and **one** skill
+  * Volunteers who have **both** NAME and SKILL keyword will be returned (i.e. `AND` search).
 
 Examples:
-* `vfind n/John` 
```

Comment:
would suggest to combine the `common cases ` and `example` sections as i feel like common cases section did not appear in other features, better to standardize ("Case 1: " also not what we used).

Maybe u can do like:
```
Examples:
- vfind n/John (search for one name)
    - returns `john` and `John Doe` (Volunteers who have that NAME keyword will be returned)
```

Reply:
idk i did it cos got some ppl in the PE dry run dumb dumb and misunderstand what vfind is about so i tot this wd make it crystal clear

Reply:
ok changed